cmd/swapcli/main.go

```diff
@@ -397,6 +414,36 @@ func runBalances(ctx *cli.Context) error {
 	return nil
 }
 
+func runETHAddress(ctx *cli.Context) error {
+	c := newRRPClient(ctx)
+	balances, err := c.Balances()
```

Comment:
I don't think it is is great that these sub commands are generating traffic to query balances, but the alternative of creating a new API that just returns addresses has the downside of more code to maintain.

Reply:
I think adding an endpoint that returns just the addresses would be pretty small and cleaner potentially, but this is fine too for now